src/main/resources/assets/galacticraft/lang/en_us.json

```
@@ -352,6 +352,7 @@
   "tooltip.galacticraft.energy_remaining": "Energy Remaining: %s",
   "tooltip.galacticraft.infinite": "Infinite",
   "tooltip.galacticraft.creative_only": "Creative Only",
+  "death.attack.gc.oilBoom": "%s tried to put out fire with a very flammable material",
```

Generally snake case is used for lang file entries (`death.attack.gc.oil_boom`).

---

also maybe use `galacticraft` instead of `gc` to keep in line with other translation keys

```suggestion
  "death.attack.galacticraft.oil_boom": "%s tried to put out fire with a very flammable material",
```

---

It should change the line directly below it then too ;)

---

fixed locally